packages/service-worker/src/low_level.ts

```diff
@@ -146,15 +148,26 @@ export class NgswCommChannel {
 
   readonly registration: Observable<ServiceWorkerRegistration>;
 
-  readonly events: Observable<TypedEvent>;
+  readonly events: Observable<TypedEvent> = new Subject();
```

Comment:
```suggestion
  private readonly _events = new Subject<TypedEvent>();
  readonly events = this._events.asObservable();
```
This would remove the need for the type assertion below